src/misc-data/tasks.ts

```diff
@@ -0,0 +1,257 @@
+export const tasks = [
+    {
+        id: 1,
+        title: "Milestone",
+        start: new Date("2014-06-02T00:00:00.000Z"),
+        end: new Date("2014-06-02T00:00:00.000Z")
+    },
+    {
+        id: 7,
+        title: "Software validation, research and implementation",
+        orderId: 0,
+        start: new Date("2014-06-02T00:00:00.000Z"),
+        end: new Date("2014-07-12T00:00:00.000Z"),
+        completionRatio: 0.45708333333333334,
+        isExpanded: true,
```

Comment:
Those examples will be just for the Angular suite, right? If so, the `isExpanded` prop should be removed. All similar data-bound component (grid, treelist, treeview), including the Gantt, track their expanded state via a callback. The built-in mechanism keeps expanded item keys and does not rely on data item props to read that data. Thus, it's redundant.